packages/discord.js/src/structures/MessageMentions.js

```diff
@@ -185,11 +190,14 @@ class MessageMentions {
   get channels() {
     if (this._channels) return this._channels;
     this._channels = new Collection();
+    const globalChannelsPatern = new RegExp(this.constructor.ChannelsPattern.source, 'g');
```

Comment:
Can we have this cached, please? I'm not a fan of recreating RegExp instances on the fly all the time. 😅